platforms/android/library/src/main/java/io/element/android/wysiwyg/EditorEditText.kt

```diff
@@ -55,7 +65,31 @@ class EditorEditText : TextInputEditText {
 
     constructor(context: Context) : super(context)
 
-    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs)
+    constructor(context: Context, attrs: AttributeSet?) : super(context, attrs) {
+        context.theme.obtainStyledAttributes(
+            attrs,
+            R.styleable.EditorEditText,
+            0, 0
+        ).apply {
+            // Default attributes
+            var bulletGapWidth =
+                applyDimension(COMPLEX_UNIT_DIP, 4f, context.resources.displayMetrics)
+            var bulletRadius =
+                applyDimension(COMPLEX_UNIT_DIP, 2f, context.resources.displayMetrics)
+
+            try {
+                bulletGapWidth =
+                    getDimension(R.styleable.EditorEditText_bulletGap, bulletGapWidth)
+                bulletRadius = getDimension(R.styleable.EditorEditText_bulletRadius, bulletRadius)
+            } finally {
+                styleConfig = StyleConfig(
+                    bulletGapWidth = bulletGapWidth,
+                    bulletRadius = bulletRadius,
+                )
+                recycle()
+            }
+        }
```

Comment:
As this might easily grow, it might be a good idea to turn it into a separate helper class when we have to re-visit it to add further customisation?

Reply:
Good point - it is a bit lengthy already. I'd agree that it's worth refactoring on the next pass.